proxy/logging/LogAccessHttp.cc

```diff
@@ -769,8 +782,11 @@ LogAccessHttp::marshal_client_req_uuid(char *buf)
     const char *uuid = (char *)Machine::instance()->uuid.getString();
     int len;
 
-    len = snprintf(str, _MAX_CRUUID_LEN, "%s-%" PRId64 "", uuid, m_http_sm->sm_id);
-    marshal_str(buf, str, round_strlen(len + 1));
+    len = snprintf(str, sizeof(str), "%s-%" PRId64 "", uuid, m_http_sm->sm_id);
```

Comment:
> On Aug 18, 2016, at 7:25 PM, Leif Hedstrom <notifications@github.com> wrote:
> 
> In proxy/logging/LogAccessHttp.cc:
> 
> > @@ -769,8 +782,11 @@ LogAccessHttp::marshal_client_req_uuid(char *buf)
> >      const char *uuid = (char *)Machine::instance()->uuid.getString();
> >      int len;
> >  
> > -    len = snprintf(str, _MAX_CRUUID_LEN, "%s-%" PRId64 "", uuid, m_http_sm->sm_id);
> > -    marshal_str(buf, str, round_strlen(len + 1));
> > +    len = snprintf(str, sizeof(str), "%s-%" PRId64 "", uuid, m_http_sm->sm_id);
> Isn't this fixed/addressed in a separate PR?
> 
Yeh I just branched this from the other PR


> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>